src/main.h

```diff
@@ -299,7 +301,7 @@ void PruneAndFlush();
 
 /** (try to) add transaction to memory pool **/
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
-                        bool* pfMissingInputs, bool fOverrideMempoolLimit=false, const CAmount nAbsurdFee=0);
+                        bool* pfMissingInputs, int64_t nTime, bool fOverrideMempoolLimit, const CAmount nAbsurdFee);
```

Comment:
This might silently coerce boolean values intended for fOverrideMempoolLimit into nTime in merging code. Better to just put nTime at the end (or before an object that won't coerce to/from it).